dexcom/device.go

```diff
@@ -179,6 +179,11 @@ func (a *AlertSetting) Validate(validator structure.Validator, existingAlertName
 }
 
 func (a *AlertSetting) validateFixedLow(validator structure.Validator) {
+	// HACK: Dexcom - snooze of 28 is invalid; use snooze of 30 instead (per Dexcom)
```

Comment:
Will this always be 28? Or could it be < 30?

Reply:
From what I've seen and what Dexcom confirmed it is only 28. This only recently started happening after a recent deploy from Dexcom.